### ISSUE TITLE
feat: Allow create task list from string

### ISSUE THREAD

**Issue by KEINOS** [KEINOS/go-todotxt#16]
## TL; DR

Implement `LoadFromStr(tasks string) (TaskList, error)` or use "io.Reader" instead of "os.File" in `LoadFromFile()`.

```go
func Example() {
	tasks, err := todo.LoadFromStr(`
		(A) Task 3
		(B) Task 1
		(C) Task 4
		Task 2
	`)
	if err != nil {
		log.Fatal(err)
	}

	tasksToday := tasks.Filter(todo.FilterHasPriority)

	fmt.Println(tasksToday.String())

	// Output:
	// (A) Task 3
	// (B) Task 1
	// (C) Task 4
}
```

## TS; DR

Currently the below 3 functions (constructors) are available to instantiate `task.TaskList` object.

1. `LoadFromFile(file *os.File) (TaskList, error)`
2. `LoadFromPath(filename string) (TaskList, error)`
3. `NewTaskList() TaskList`

Creating a task list as below, like the one in the example of custom sort in the README, filters doesn't work.

```go
func Example() {
	tasks := todo.TaskList{
		// These sets to Todo field but other fields are empty
		// due to not parsing the Todo text.
		todo.Task{Todo: "(A) Task 3"},
		todo.Task{Todo: "(B) Task 1"},
		todo.Task{Todo: "(C) Task 4"},
		todo.Task{Todo: "Task 2"},
	}

	tasksToday := tasks.Filter(todo.FilterHasPriority)

	fmt.Println(tasksToday.String()) // tasksToday.Todo is nil

	// Output:
	// (A) Task 3
	// (B) Task 1
	// (C) Task 4
}
```
```shellsession
$ go test .
=== RUN   Example
--- FAIL: Example (0.00s)
got:

want:
(A) Task 3
(B) Task 1
(C) Task 4
FAIL
```

This is because the tasks at "Todo" field are not parsed due to the direct assignment.

### Problem

The example in the README doesn't work as it says. It has typo and non-existing function.

We must update the README, but to avoid further mistakes leading to "Go Playground" is a good idea before leading to the `godoc` page.

The problem is that "Go Playground" can not have external text file to read.
